plotting_refactor/FitPage.py

```diff
@@ -0,0 +1,33 @@
+from PySide6 import QtWidgets
+from FitPageUI import Ui_fitPageWidget
+
+
+class FitPage(QtWidgets.QWidget, Ui_fitPageWidget):
+    def __init__(self, int_identifier):
+        super(FitPage, self).__init__()
+        self.setupUi(self)
+
+        #fitPageIdentifier keeps track of which number this fitpage is identifier by (it is incremental)
+        self.fitPageIdentifier = int_identifier
+
+        self.comboBoxFormFactor.addItems(["Sphere", "Cylinder"])
+        self.doubleSpinBox_height.setDisabled(True)
+        self.comboBoxFormFactor.currentIndexChanged.connect(self.index_changed)
+
+    def get_int_identifier(self):
+        return self.fitPageIdentifier
+
+    def get_combobox_index(self):
+        return self.comboBoxFormFactor.currentIndex()
+
+    def get_checkbox_fit(self):
+        return self.checkBoxCreateFit.isChecked()
+
+    def get_checkbox_2d(self):
+        return self.checkBox2dData.isChecked()
+
+    def index_changed(self, selected_item):
+        if selected_item == 0:
```

Comment:
My only other comment: Python uses 'truthiness' of objects. 0 is false and 1 is true, so, unless there is a chance set_item could be something other than 0 or 1, this if/else could simplify to `self.doubleSpinBox_height.setDisabled(not selected_item)`.